Dockerfile

```
@@ -0,0 +1,14 @@
+FROM itkdev/php8.3-fpm:latest
+
+USER root
+
+# Add rsync
+RUN apt-get update && apt-get --yes install rsync && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Add node
+RUN apt-get -y install curl gnupg
```

Be consistent with the rest of the file:

```suggestion
RUN apt-get --yes install curl gnupg
```